.github/ISSUE_TEMPLATE/new_score.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: New Score or Metric
-about: Suggest a new score or metric which could be added to the repository
+about: Suggest a new metric or statistical test which could be added to the repository
```

Comment:
Do we want to say "tool" or "utility" as well? E.g. a new pre-processing tool

Reply:
I am happy to. 

I wasn't sure if tools or utilities would be added on their own, or always as part of a new metric or statistical test. Also, I didn't think all of the contributing guide or pull request checklist necessarily applied to tools - but people can just ignore the parts that don't apply.

But it sounds like those concerns aren't warranted, so, yes I am happy to add that.

Reply:
Is it better to just say "tool", or "data processing tool"?

Reply:
Shorter probably better in this context. I will go with "tool" unless anyone objects (very happy to change it though)